src/poromultiphase_scatra/4C_poromultiphase_scatra_function.cpp

```diff
@@ -115,22 +198,129 @@ void PoroMultiPhaseScaTra::add_valid_poro_functions(Core::Utils::FunctionManager
+          parameter<int>("NUMPARAMS"),
+          group("PARAMS",
+              {
+                  parameter<double>("n"),
+                  parameter<double>("Pb50"),
+                  parameter<double>("CaO2_max"),
+                  parameter<double>("alpha_bl_eff"),
+                  parameter<double>("gamma_rho_SV"),
+                  parameter<double>("rho_oxy"),
+                  parameter<double>("rho_IF"),
+                  parameter<double>("rho_bl"),
+                  parameter<double>("alpha_IF"),
+              }),
+      }),
+      all_of({
+          selection<std::string>(
+              "POROMULTIPHASESCATRA_FUNCTION", {"OXYGEN_TRANSVASCULAR_EXCHANGE_LAW_DISC"}),
+          parameter<int>("NUMPARAMS"),
+          group("PARAMS",
+              {
+                  parameter<double>("n"),
+                  parameter<double>("Pb50"),
+                  parameter<double>("CaO2_max"),
+                  parameter<double>("alpha_bl_eff"),
+                  parameter<double>("gamma_rho"),
+                  parameter<double>("rho_oxy"),
+                  parameter<double>("rho_IF"),
+                  parameter<double>("rho_bl"),
+                  parameter<double>("S2_max"),
+                  parameter<double>("alpha_IF"),
+              }),
+      }),
+      all_of({
+          selection<std::string>("POROMULTIPHASESCATRA_FUNCTION", {"LUNG_OXYGEN_EXCHANGE_LAW"}),
+          parameter<int>("NUMPARAMS"),
+          group("PARAMS",
+              {
+                  parameter<double>("rho_oxy"),
+                  parameter<double>("DiffAdVTLC"),
+                  parameter<double>("alpha_oxy"),
+                  parameter<double>("rho_air"),
+                  parameter<double>("rho_bl"),
+                  parameter<double>("n"),
+                  parameter<double>("P_oB50"),
+                  parameter<double>("NC_Hb"),
+                  parameter<double>("P_atmospheric"),
+                  parameter<double>("volfrac_blood_ref"),
+              }),
+      }),
+      all_of({
+          selection<std::string>(
+              "POROMULTIPHASESCATRA_FUNCTION", {"LUNG_CARBONDIOXIDE_EXCHANGE_LAW"}),
+          parameter<int>("NUMPARAMS"),
+          group("PARAMS",
+              {
+                  parameter<double>("rho_CO2"),
+                  parameter<double>("DiffsolAdVTLC"),
+                  parameter<double>("pH"),
+                  parameter<double>("rho_air"),
+                  parameter<double>("rho_bl"),
+                  parameter<double>("rho_oxy"),
+                  parameter<double>("n"),
+                  parameter<double>("P_oB50"),
+                  parameter<double>("C_Hb"),
+                  parameter<double>("NC_Hb"),
+                  parameter<double>("alpha_oxy"),
+                  parameter<double>("P_atmospheric"),
+                  parameter<double>("ScalingFormmHg"),
+                  parameter<double>("volfrac_blood_ref"),
+              }),
+      })});
```

Comment:
```suggestion
      }),});
```

Minor: this comma will move the `one_of` at the top to a separate level. I'll leave it to you whether this is prettier.